sysconfig/ubuntu-18.04/modules.yaml

```diff
@@ -0,0 +1,38 @@
+modules:
+  enable::
+      - tcl
+  prefix_inspections:
+    bin:
+      - PATH
+    bin64:
+      - PATH
+    share/man:
+      - MANPATH
+    man:
+      - MANPATH
+    lib:
+      - LD_LIBRARY_PATH
+    lib64:
+      - LD_LIBRARY_PATH
+    lib/pkgconfig:
+      - PKG_CONFIG_PATH
+    lib64/pkgconfig:
+      - PKG_CONFIG_PATH
+    '':
+      - CMAKE_PREFIX_PATH
+  tcl:
+    all:
+      filter:
+        environment_blacklist: ['CPATH', 'LIBRARY_PATH']
+      suffixes:
+          '+profile': 'profile'
+    naming_scheme: '${PACKAGE}/${VERSION}'
+    hash_length: 0
+    whitelist:
+      - neuron
+      - neuronmodels
+      - mvapich2
+      - tau
+    blacklist:
```

Comment:
May be we can add all our packages in bbp applications here?

Reply:
Yes